ui/packages/consul-ui/app/components/menu/README.mdx

```diff
@@ -0,0 +1,52 @@
+# Menu
+
+A component use for menu systems with the correct aria attributes applied.
+Internally uses our `{{aria-menu}}` modifier for aria keyboarding.
+
+Additionally it is made to work in tandem with the `<Disclosure />` component if
+required (a relatively common usecase)
+
+This component should not be used for top site navigation, but it should be used
+for menus within the top site navigation for choosing options, for example
+choosing a namespace or partition etc.
+
+```hbs preview-template
+<Menu as |menu|>
+  <menu.Item>
+    <menu.Action>Item 1</menu.Action>
+  </menu.Item>
+  <menu.Separator />
+  <menu.Item>
+    <menu.Action>Item 2</menu.Action>
+  </menu.Item>
+  <menu.Separator>
```

Comment:
I'm having trouble finding an example of an a11y separator with text.  This might not function as a title as the example code here indicates.  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role

Should there be another component to create a titled group of menu items?

Reply:
Yeah I'm not totally sure to be fair, but this is how it is already all over the UI. I'm guessing a separator can be a piece of text (especially as MDN mentions you can give them an aria-label, which would be if you couldn't add screen-readable text inside)

> Should there be another component to create a titled group of menu items?

Considering the above I'm happy using role="separator"s with or without text/aria-labels.